vendor/bat-native-ads/src/bat/ads/internal/container_util_unittest.cc

```diff
@@ -0,0 +1,286 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/container_util.h"
+
+#include "testing/gtest/include/gtest/gtest.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+
+TEST(BatAdsContainerUtilTest,
+    SplitVectorIntoSingleChunk) {
+  // Arrange
+  const std::vector<std::string> vector = {
+    "item 1",
+    "item 2",
+    "item 3",
+    "item 4",
+    "item 5",
+    "item 6"
+  };
+
+  // Act
+  const std::vector<std::vector<std::string>> vectors = SplitVector(vector, 6);
+
+  // Assert
+  const std::vector<std::string> expected_vector = vector;
```

Comment:
I'm aware that it's kinda weird to receive random comments on an already merged PR, but since I've already lurked recent changes...

I think you meant something like
```
std::vector<std::vector<std::string>> expected_vectors = { vector };
EXPECT_EQ(expected_vectors, vectors);
```
, or whatever the coding style is.

The current implementation doesn't really test the actual `SplitVector` call - `vectors` is unused and we only validate that `vector == vector`, which is obviously true.

Same applies to a few cases below.

Reply:
Thanks, fixed at https://github.com/brave/brave-core/pull/8072